etc/features.yaml

```diff
@@ -85,4 +85,6 @@
     inspec-reporter-progress-bar:
       description: Use progress bar streaming reporter
     inspec-reporter-child-status:
-      description: Child status reporter used in inspec parallel reporting.
\ No newline at end of file
+      description: Child status reporter used in inspec parallel reporting.
+    inspec-profile-mandatory-signing:
+      flag: CHEF_PREVIEW_MANDATORY_PROFILE_SIGNING
```

Comment:
I'm wondering about the word "flag". Can we do this instead:
```
inspec-my-new-feature:
  # This will cause us to be sensitive to CHEF_PREVIEW_MY_NEW_FEATURE
  env_preview: true
  ```
  
 Then later you can do something like:
 ```
   if feat.previewable? do
     yield
    end
 ```
 
 and then in Feature.previewable?
 check to see if feat.env_preview is enabled (if false, we're not in preview, so return true)
 if the feature does support env preview, then generate the associated ENV var name and check for it, and return true if it is set.